OWNERS_ALIASES

```diff
@@ -18,6 +18,7 @@ aliases:
     - jackfrancis
     - mboersma
     - nojnhuh
+    - Jont828
   cluster-api-azure-reviewers:
     - Jont828
```

Comment:
Should we remove me from reviewers in that case?

Reply:
Good catch, that seems to be the pattern, I'll update